### PR TITLE
#165305729 Follow and Unfollow Users

### PR DESCRIPTION
**Pull request by matthenge**
#### What does this PR do?
- Allows a user to follow or unfollow another user.
#### Description of Task to be completed?
- Add follow functionality
- Add unfollow functionality
- Add get the list of followers
- Add get the list of users a user is following
#### How should this be manually tested?
After cloning the repo:
```
$ git checkout ft-follow-users-165305729
$ npm npm run start:dev
```
#### What are the relevant pivotal tracker stories?
[#165305729](https://www.pivotaltracker.com/story/show/165305729)

### CODE REVIEW

**Review by Bakley — APPROVED** [review]

File: src/components/profileNavbar.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Navbar, Nav, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import '../styles/App.css';
+
```
Comment (IssaIan) [inline src/components/profileNavbar.jsx]: Rebase correctly...no profile navbar in develop.

**Review by IssaIan — CHANGES_REQUESTED** [review]

File: src/components/profileNavbar.jsx (hunk 1 above)
Comment (matthenge) [reply src/components/profileNavbar.jsx]: @IssaIan  already fixed

**Review by a-braham — APPROVED** [review]
LGTM

**Review by Ogutu-Brian — APPROVED** [review]

**Review by Ogutu-Brian — APPROVED** [review]

**Review by IssaIan — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -20,7 +20,11 @@
   "bugs": {
     "url": "https://github.com/andela/ah-the-immortals-frontend/issues"
   },
-  "jest": {},
+  "jest": {
```
Comment (koechkevin) [inline package.json]: Why? we already have it in jest.config.js

**Review by koechkevin — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (matthenge) [reply package.json]: I have fixed it. It was a change in @a-braham Pull request that I had cherry-picked

**Review by koechkevin — APPROVED** [review]

**Pull request merged**